config/jacoco.gradle

```diff
@@ -29,7 +29,7 @@ task jacocoTestReport(type: JacocoReport, dependsOn: "testStagingDebugUnitTest")
     def fileFilter = fileGenerated + packagesExcluded
 
     classDirectories = fileTree(
-        dir: "$buildDir/intermediates/classes/staging/debug",
+        dir: "$buildDir/intermediates/javac/stagingDebug/compileStagingDebugJavaWithJavac/classes",
```

Comment:
jacoco has been pre-configured before, androidx migration doesn't do anything related to this config change, it may wait for a long time to be merged inclusively

Reply:
@sleepylee PR is separated #38 🙏